tests/08-native-runs/14-sha-256.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -e
+
+./run-one.sh 14-sha-256
```

Comment:
A fairly recent change is that it now also requires a line in `tests/08-native-runs/Makefile` for the test to be compiled and run. (Can copy the 13-coffee line and modify.)